File: Convex/Figure/Void.cs
namespace Convex.Figure
{
    public class Void: IFigure
    {
        public IFigure Add(Geometry.Point point)
        {
            return new Point(point);
        }

        public double Area
        {
            get => 0;
        }

        public double Perimeter
        {
            get => 0;
        }
    }
}

File: src/convex/figure/Void.java
package convex.figure;

import convex.geometry.Point;

public class Void implements Figure {
    public Figure add(Point p, Point task) {
        return new convex.figure.Point(p, task);
    }
    @Override
    public double perimeter() {
        return 0;
    }
    @Override
    public double area() {
        return 0;
    }
    @Override
    public double result() {
        return 0;
    }
}